src/main/java/com/urbancode/jenkins/plugins/ucdeploy/VersionHelper.java

```diff
@@ -295,6 +295,8 @@ private void setComponentVersionProperties(
                 propSheetDef = compClient.getComponentVersionPropSheetDef(component);
                 propSheetDefId = (String) propSheetDef.get("id");
                 propSheetDefPath = (String) propSheetDef.get("path");
+                propSheetDefPath = propSheetDefPath.replaceAll("/", "&");
```

Comment:
P.P.S. I get exactly the same results running the test.groovy script on my windows 7 machine with groovy 2.4.10. with the UDRestClient jar you supplied.

Mitch